website/thaliawebsite/api/v1/cleaned_model_serializer.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+
+
+class CleanedModelSerializer(serializers.ModelSerializer):
+    """Custom ModelSerializer that explicitly clean's a model before it is saved."""
+
+    def validate(self, attrs):
```

Comment:
I am not sure we should add this. v1 has been running a long time and this might break things.

Reply:
Well if it breaks stuff, it breaks on adding stuff that doesn't `clean`. 

What I've learned from this bug (and that it has been here for a long time without us noticing in fact) is that we rely a lot on front end logic. I really think we should clean in v1 too!